pkg/oam/util/helper.go

```diff
@@ -197,3 +202,8 @@ func Object2Map(obj interface{}) (map[string]interface{}, error) {
 	err = json.Unmarshal(bts, &res)
 	return res, err
 }
+
+// GenTraitName generate trait name
```

Comment:
please add a unit test for this function

Reply:
ok